manim/animation/fading.py

```diff
@@ -1,13 +1,63 @@
-"""Fading in and out of view."""
+"""Fading in and out of view.
```

Comment:
Why is this example so large? Since these examples are written in the code they should be as condensed and illustrative as possible.

Reply:
It is so large because the source code is hidden to the user.
The functions are explained on the display and as you said, they actually are as condensed as possible and all visible at one glance.  
You can see a rendered example here: https://manimce--827.org.readthedocs.build/en/827/reference/manim.animation.fading.html
And a similar approach already exists here:
https://docs.manim.community/en/latest/reference/manim.animation.creation.html